### ISSUE TITLE
Not receiving all MMS

### ISSUE THREAD

**Issue by tiffanytkn** [moezbhatti/qksms#1526]
I haven't been receiving some MMS in some group chats I am part of.

I began noticing this difference when I upgraded to Version 3.7.6 (2205) on F-Droid around November 17.

I found out from another person in those chats who sent me screenshots from his phone and others' comments on photos that I had never received.

I do receive some MMS, but not all. The senders I didn't receive messages from use Android phones and iPhones, if that makes a difference.

When I switched back to the default messaging app on my OnePlus 5T, I saw the option to download pending MMS from various contacts who'd messaged group chats. Most of those messages then did come through.

**Comment by ds6** [comment]
This has happened to me as well. When I sync the database I can see these suspicious entries:

![Screenshot_20191207-143040_QKSMS_1](https://user-images.githubusercontent.com/16788697/70380334-166a1700-18ff-11ea-9d34-de95a3c8bf57.png)
![Screenshot_20191207-142933_QKSMS_1](https://user-images.githubusercontent.com/16788697/70380336-1bc76180-18ff-11ea-9745-0f37c4f6a190.png)

In both instances I was told I was sent a message and never receieved it. Only after syncing did these pop up.

**Comment by moezbhatti** [comment]
Very interesting. @ds6, if you switch to another SMS app, will it give you an option to download these messages? Or perhaps it might just download them automatically and then show their contents

**Comment by ds6** [comment]
I first noticed this when I switched back to the default SMS app to see if it was QKSMS. They weren't there, so I turned on QKSMS again and tried re-syncing the database. I saw the empty messages then.
Don't know if this is related, but after I moved to my new phone and manually copied my message databases from /data/data, I don't see the MMS images in QKSMS from that period, but they appear in the stock app fine. Newly receieved images (except these missing ones) show up fine.
